dialect/sql/schema/mysql.go

```diff
@@ -449,6 +435,39 @@ func (d *MySQL) tableSchema() sql.Querier {
 	return sql.Raw("(SELECT DATABASE())")
 }
 
+// columnData returns column data: parts, size and signedness
+func columnData(typ string) ([]string, int64, bool, error) {
```

Comment:
`parseColumn` is more meaningful IMO. 
```suggestion
func parseColumn(typ string) ([]string, int64, bool, error) {
```